README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+Hello, World! My name is Kwabene Kalumbula.
+
+For this, my first assignment in MACS 30500: Computing for the Social Sciences, I have been instructed to provide a short biograpghy. Thus, as the leader of a band of cartoon scallions once said: "Allow [me] to introduce [myself]." ![image of scallion](https://i.kym-cdn.com/entries/icons/original/000/027/174/veggie.jpg)
+
+#### Education
```

Comment:
I love how you create different headers for different sections of your introduction, it makes it much easier to read and digest the information!